src/main/java/com/google/javascript/gents/RemoveGoogScopePass.java

```diff
@@ -113,7 +113,14 @@ private Node maybeRecordAndRemoveAlias(Node assign) {
   }
 
   private void maybeReasignAlias(Node assign) {
-    String alias = assign.getFirstFirstChild().getQualifiedName();
+    // Find the name of the deepest first child.
+    String alias = null;
+    for (Node child = assign.getFirstChild(); child != null; child = child.getFirstChild()) {
+      if (child.isName()) {
+        alias = child.getString();
```

Comment:
`getQualifiedName()` calls `getString()`, so they are equivalent, but switching for consistency.